templates/user/edit-profile.html

```diff
@@ -148,7 +148,13 @@
                                 <td colspan="2"><span class="inline-header">
                                     {% if profile.is_totp_enabled %}
                                         {{ _('Two Factor Authentication is enabled.') }}
-                                        <a href="{{ url('disable_2fa') }}" class="button inline-button">Disable</a>
+					{% if request.user.is_superuser %}
+					    <a href="{{ url('disable_2fa') }}" class="button inline-button">Disable</a>
+					{% else %}
+					    <a href="javascript:alert('{{ _('Contact an admin to disable Two Factor Authentication')|escapejs }}')"
+					       title="{{ _('Contact an admin to disable Two Factor Authentication') }}" 
```

Comment:
Probably better to have a more descriptive message like `The administrators for this site require all staff to have two-factor authentication enabled, so it may not be disabled at this time.`